### PR TITLE
Hero Image Added

### PR DESCRIPTION
**Pull request by askprateek**

### CODE REVIEW

**Comment by handasolo** [comment]
Um, looks good but I don't like the image effect. Try something better?

File: src/client/css/style.css [hunk 1]
```diff
@@ -172,6 +172,48 @@ ul {
     width: auto;
   }
 }
+/*Hero Image Section */
+.animated {
+  -webkit-animation-duration: 2s;
```
Comment (handasolo) [inline src/client/css/style.css:190]: add `-moz-` and `-o-` too

File: src/client/css/style.css [hunk 2]
```diff
@@ -172,6 +172,48 @@ ul {
...
+  animation-duration: 2s;
+  -webkit-animation-fill-mode: both;
```
Comment (handasolo) [inline src/client/css/style.css:195]: same here.

File: src/client/css/style.css [hunk 3]
```diff
@@ -172,6 +172,48 @@ ul {
...
+  animation-fill-mode: both;
+}
+
+@-webkit-keyframes fadeInDown {
+  0% {
+    opacity: 0;
+    -webkit-transform: translate3d(0, -100%, 0);
```
Comment (handasolo) [inline src/client/css/style.css:204]: `-moz-`,`-o-` and `-ms-`

File: src/client/css/style.css [hunk 4]
```diff
@@ -172,6 +172,48 @@ ul {
...
+    transform: translate3d(0, -100%, 0);
+  }
+
+  100% {
+    opacity: 1;
+    -webkit-transform: none;
```
Comment (handasolo) [inline src/client/css/style.css:213]: same

File: src/client/css/style.css [hunk 5]
```diff
@@ -172,6 +172,48 @@ ul {
...
+    transform: none;
+  }
+}
+
+@keyframes fadeInDown {
+  0% {
+    opacity: 0;
+    -webkit-transform: translate3d(0, -100%, 0);
```
Comment (handasolo) [inline src/client/css/style.css:224]: same with this.

File: src/client/css/style.css [hunk 6]
```diff
@@ -172,6 +172,48 @@ ul {
...
+    transform: translate3d(0, -100%, 0);
+  }
+
+  100% {
+    opacity: 1;
+    -webkit-transform: none;
```
Comment (handasolo) [inline src/client/css/style.css:233]: same with this

File: src/client/css/style.css [hunk 7]
```diff
@@ -172,6 +172,48 @@ ul {
...
+    transform: none;
+  }
+}
+
+.fadeInDown {
+  -webkit-animation-name: fadeInDown;
```
Comment (handasolo) [inline src/client/css/style.css:242]: Only `-moz-` and `-o` for this.

File: src/client/index.html [hunk 8]
```diff
@@ -44,6 +44,16 @@
       <!-- Sticky Events section goes here -->
 
       <!-- Hero Image goes here -->
+      <section id="Hero_Image">
+        <div class ="row">
+          <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12" style="position: relative;">
+            <img src="img/hero/back.png" class ="img-responsive animated fadeInDown" style="position: relative; top: 0; left: 0;max-width:100%;"/>
+            <img src=img/hero/logo.png class="img-responsive col-xs-2 col-sm-2 col-md-2 col-lg-2 animated fadeInDown" style="position: absolute; bottom:5px;" />
```
Comment (handasolo) [inline src/client/index.html]: I see inline styles. Any reason for them?

Comment (askprateek) [inline src/client/index.html]: Just to achieve the desired effect

Comment (handasolo) [inline src/client/index.html]: I meant what are they doing in the html file instead of the css file xD

Comment (askprateek) [inline src/client/index.html]: :P

**Comment by handasolo** [comment]
Checking for any possible changes one last time.

**Comment by handasolo** [comment]
Merging. Thank you for your patience. :)

**Pull request merged**